app/utils/elastic.py

```diff
@@ -0,0 +1,61 @@
+import requests
+import json
+import os
+from elasticsearch import Elasticsearch
+from dotenv import load_dotenv
+
+load_dotenv()
+
+ELASTIC_SERVICE_URL=os.environ.get("ELASTIC_SERVICE_URL")
+ELASTICSEARCH_USER=os.environ.get("ELASTICSEARCH_USER")
+ELASTICSEARCH_PASS=os.environ.get("ELASTICSEARCH_PASS")
+
+client = Elasticsearch(
+    ELASTIC_SERVICE_URL,
+    basic_auth=(ELASTICSEARCH_USER, ELASTICSEARCH_PASS),
+    verify_certs=False
+)
+
+def find_elastic_dataset_matches(title):
+    payload = {
+        "match": {
+            "shortTitle": {
+                "query": title
+            }
+        }
+    }
```

Comment:
We could update this query to do something more fancy: fuzzy matching, match on multiple fields etc